docs/latest/getting-started/fetching-data.md

```diff
@@ -9,56 +9,53 @@ during rendering. In real projects, this is often different. In many cases you
 may need to read a file from disk (e.g. markdown for a blog post), or fetch some
 user data from an API or database.
 
-These operations are all asynchronous. Rendering however, is always synchronous.
-Instead of fetching data directly during rendering, it should be loaded in a
-route's `handler` function and then passed to the page component via first
-argument to `ctx.render()`.
+These operations are all asynchronous. However, rendering is always synchronous.
+So, to fetch data dynamically, we can use our brand new `async-route`
+components, which abstract the old way of fetching data by writing the
+`Handlers`.
 
-The data that is passed to `ctx.render()` can then be accessed via the
-`props.data` field on the page component.
+To fetch our data, we must first make our route component asynchronous and then
+create a way to fetch the data, for example with the `fetch` API. The `req`
+parameter is mandatory, and it indicates that we will fetch some data. Then we
+need the `ctx` prop to get our route parameters (in this case, it's the
+username).
```

Comment:
This page is not a guide on how to refactor the old handlers to new async route components. Can you rewrite this so it doesn't sound like one?

Reply:
Thank you! Does this sound better? 

"
In order to fetch data, the route component must be asynchronous. The `req`
parameter is mandatory, and it indicates that certain data will be fetched. The
second `req` parameter is used to get the route parameters, such as the username
is this case.
"